charts/function-mesh-operator/templates/controller-manager-rbac.yaml

```diff
@@ -154,6 +154,15 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
```

Comment:
could you please help to verify if `leases/finalizers` needs as well for OpenShift platform?